content/hi/docs/concepts/architecture/leases.md

```diff
@@ -0,0 +1,103 @@
+---
+title: पट्टों
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+वितरित प्रणालियों को अक्सर _पट्टों_ की आवश्यकता होती है, जो साझा संसाधनों को लॉक करने के लिए एक तंत्र प्रदान करते हैं
+और एक सेट के सदस्यों के बीच गतिविधि का समन्वय करें।
+कुबेरनेट्स में, पट्टे की अवधारणा को [लीज](/docs/reference/kubernetes-api/cluster-resources/lease-v1/) द्वारा दर्शाया गया है।
+`coordination.k8s.io` में ऑब्जेक्ट्स {{< glosary_tooltip text="API Group" term_id="api-group" >}},
+जिनका उपयोग सिस्टम-महत्वपूर्ण क्षमताओं जैसे नोड दिल की धड़कन और घटक-स्तरीय नेता चुनाव के लिए किया जाता है।
```

Comment:
here हर्टबीट  and लीडर  seems more appropriate.
जिनका उपयोग सिस्टम-महत्वपूर्ण क्षमताओं जैसे नोड हर्टबीट और घटक-स्तरीय लीडर चुनाव के लिए किया जाता है।